app/__init__.py

```diff
@@ -1,14 +1,22 @@
 """ Application """
-import os
-from flask import Flask
-from config import config
-from models import connect_db
+# import os
+from flask import Flask, render_template
+from config import Config
+from .recipe import get_random_joke, get_random_recipes
+# from models import connect_db
 
 app = Flask(__name__)
 
 def create_app(config_name):
-    app.config.from_object(config[config_name])
-    config[config_name].init_app(app)
-    connect_db(app)
+    app.config.from_object(Config[config_name])
+    Config[config_name].init_app(app)
```

Comment:
@Huulamnguyen what are we trying to implement on line 11 and 12 ? This implementation seems to be crashing the app. Kindly look into it.